### PR TITLE
Add basic simulator status bar

### PR DESCRIPTION
**Pull request by dougajmcdonald**
Add's a status bar to the simulator to show whether it's running, how many warriors are loaded etc.

### CODE REVIEW

File: src/containers/simulator/simulatorStatus.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      outcome += ` - winner(${roundResult.winnerId})`
+    }
+    return outcome;
+  }
+
+  if(parseResults.length > 0 && !isInitialised && !isRunning) {
+    return `${parseResults.length} warriors loaded, awaiting init command`
+  }
+
+  if(parseResults.length > 0 && isInitialised && !isRunning) {
```
Comment (gareththegeek) [inline src/containers/simulator/simulatorStatus.js]: Almost the same if condition twice in a row, could you combine them?

Comment (dougajmcdonald) [reply src/containers/simulator/simulatorStatus.js]: Yeah, this bit is shit. non refactored rubbish just to see if is a sensible concept or not.

The logic is also floored, can you spot where?

Comment (gareththegeek) [reply src/containers/simulator/simulatorStatus.js]: Floored or flawed?

Comment (gareththegeek) [reply src/containers/simulator/simulatorStatus.js]: No, I can't :P

**Pull request merged**